.github/workflows/precommit.yml

```
@@ -466,6 +466,64 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  combine-coverage:
+    if: ${{ success() || failure() }}
+    name: Combine coverage
+    needs: test
+    runs-on: ubuntu-latest-64-cores
```

```suggestion
    runs-on: ubuntu-latest
```
No need to use the strong machine for this